web/static/js/main.js

```diff
@@ -0,0 +1,10 @@
+function selectTheme(checkState)
```

Comment:
Perhaps a better name than checkState. What does the truthiness of this parameter mean?

Reply:
Changed name from ambiguous "checkState" which refers to the state of the UI element, to "darkThemeSelected" which better indicates what the boolean value semantically means.